POC/GPT4V-OCR.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Again, just add scaling pre-processing to the image reading, or rescale the images once and append to filenames.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>